src/mixins/filing-template-mixin.ts

```diff
@@ -206,7 +206,12 @@ export default class FilingTemplateMixin extends DateMixin {
         date: this.getCurrentDate, // "absolute day" (YYYY-MM-DD in Pacific time)
         folioNumber: this.getFolioNumber // business folio number, unless overridden below
       },
-      business: this.getEntitySnapshot.businessInfo,
+      business: {
+        foundingDate: this.getEntitySnapshot.businessInfo.foundingDate,
```

Comment:
If I'm not mistaken, we only need this in IAs and Registrations. After my investigation and with the help of Argus:
- business.legalType is needed everywhere
- business.foundingDate is needed in IAs and Registrations
- business.identifier is needed in IAs and Registrations
- business.legalName is needed in Alterations

I'll double check with Argus regarding that.

Reply:
@severinbeauvais Sorry, I deleted my previous comment to change it's position. No, there shouldn't be any problem to set all 4 properties!